yaml/xpu_functions.yaml

```diff
@@ -228,7 +215,7 @@ supported:
   - min
   - min.unary_out
   - min.dim_min
-  - sum.out
+  - sum
```

Comment:
You are saying it? It should not be registered? It's dispatched by  `CompositeExplicitAutograd` 
<img width="439" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/eac00d98-445d-4979-b04a-f90a8a34393b">